spec/swagger_helper.rb

```diff
@@ -4,12 +4,17 @@
 require 'webmock/rspec'
 require './spec/api/v1/openapi'
 require './spec/api/v1/schemas/util'
+require './spec/api/v2/openapi'
 
-include Api::V1::Schemas::Util # rubocop:disable Style/MixinUsage
+# rubocop:disable Style/MixinUsage
+include Api::V1::Schemas::Util
+include Api::V2::Schemas::Util
```

Comment:
Rubocop's complaints are real here, what methods are needed from these modules in the current context and why can't they be called statically?